omise-wp-admin.php

```diff
@@ -204,7 +233,36 @@ function register_dashboard_script() {
 			wp_enqueue_script( 'omise-dashboard-js', plugins_url( '/assets/javascripts/omise-dashboard-handler.js', __FILE__ ), array( 'jquery' ), OMISE_WOOCOMMERCE_PLUGIN_VERSION, true );
 			wp_enqueue_style( 'omise-css', plugins_url( '/assets/css/omise-css.css', __FILE__ ), array(), OMISE_WOOCOMMERCE_PLUGIN_VERSION );
 		}
+
+		function list_charges() {
```

Comment:
oops, I forgot that that `list()` is a PHP's reserved word, have to use other name 😅